### PR TITLE
Minor bug fix and feature extension

### LINKED ISSUE #32: Implement initializing multiple fins instead of single set
**Is your feature request related to a problem? Please describe.**

A feature to add multiple fins for complex rocket designs.

**Describe the solution you'd like**

Instead of taking an object of fin schema in the POST rocket route, take an array of fin objects and add them to new/existing rocket object. Just like how it's currently being implemented for parachutes.

### PR DESCRIPTION
**Pull request by GabrielBarberini**
- Fixed airfoil signature to be the same as power_off_drag (str -> List[tuples])
- Implement multiple rocket fins attachment

Fixes #32

## Summary by CodeRabbit

- **New Features**
	- Enhanced representation of airfoil data in the Fins class, now supporting complex structures.
	- The Rocket class can now accommodate multiple Fins instances, improving flexibility in rocket design.
	- Introduced a new method for converting a list of Fins objects into RocketPyFins, enhancing the service's functionality.

- **Bug Fixes**
	- Adjusted handling of fins in the RocketService for improved processing of multiple fin sets.

- **Documentation**
	- Updated type annotations for better clarity and type safety.

### CODE REVIEW

**Comment by GabrielBarberini** [comment]
@Gui-FernandesBR can you please confirm if I need to evaluate static margin for each `rocketpy_rocket.aerodynamic_surfaces.add` or can I evaluate it only after adding everything (out of the loop) ?

File: lib/services/rocket.py [hunk 1]
```diff
@@ -134,6 +137,18 @@ def get_rocketpy_nose(nose: NoseCone) -> RocketPyNoseCone:
...
+    def get_rocketpy_finset_list_from_fins_list(
+        cls, fins_list: List[Fins]
+    ) -> List[RocketPyFins]:
+        rocketpy_finset_list = []
+        if not fins_list:
+            return []
+        for fins in fins_list:
+            rocketpy_fins = cls.get_rocketpy_finset(fins, fins.fins_kind)
+            rocketpy_finset_list.append(rocketpy_fins)
+        return rocketpy_finset_list
```
Comment (Gui-FernandesBR) [inline lib/services/rocket.py:150]: noblock: you could use list comprehension here to become more readable.

File: lib/services/rocket.py [hunk 2]
```diff
@@ -63,9 +63,12 @@ def from_rocket_model(cls, rocket: Rocket) -> Self:
...
         # FinSet
-        finset = cls.get_rocketpy_finset(rocket.fins, rocket.fins.fins_kind)
-        rocketpy_rocket.aerodynamic_surfaces.add(finset, finset.position)
-        rocketpy_rocket.evaluate_static_margin()
+        rocketpy_finset_list = cls.get_rocketpy_finset_list_from_fins_list(
+            rocket.fins
+        )
+        for finset in rocketpy_finset_list:
+            rocketpy_rocket.aerodynamic_surfaces.add(finset, finset.position)
+            rocketpy_rocket.evaluate_static_margin()
```
Comment (Gui-FernandesBR) [inline lib/services/rocket.py:71]: The `evaluate_static_margin()` is really time-consuming, you only need to call it once, after all the fins have already been set.

**Review by Gui-FernandesBR — APPROVED** [review]
Looking good.

File: lib/services/rocket.py (hunk 2 above)
Comment (GabrielBarberini) [reply lib/services/rocket.py]: patched in [95aaf32](https://github.com/RocketPy-Team/Infinity-API/pull/33/commits/95aaf329a65646f9940480bee6b370ef71bc3059)

File: lib/services/rocket.py (hunk 1 above)
Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: Thanks for the review!

I could now think in a way to use list comprehension here, could you please share what you thought?

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: Any clue @coderabbitai ?

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: and what about `rocketpy_finset_list.append(rocketpy_fins)` ?

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: and what about rocketpy_finset_list.append(rocketpy_fins) https://github.com/coderabbitai ?

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: and what about `rocketpy_finset_list.append(rocketpy_fins)` @coderabbitai ?

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: Right in front of me 🤦🏻 , thanks!

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: Can you suggest that commit, please? @coderabbitai

Comment (GabrielBarberini) [reply lib/services/rocket.py:150]: I mean, can you commit it please?

Comment (GabrielBarberini) [reply lib/services/rocket.py]: patched in [dc0138f](https://github.com/RocketPy-Team/Infinity-API/pull/33/commits/dc0138f7a7274eb498e0db8823d75be3cde2ce6d)

**Review by Gui-FernandesBR — APPROVED** [review]

**Pull request merged**